wp-content/themes/bh/scss/donate.scss

```diff
@@ -400,104 +415,83 @@ h1 {
-}
 
-#donate-process-thank-you {
-  p {
-    font-size: 22px;
-    line-height: 46px;
+  input[type=checkbox],
+  input[type=radio   ] {
+    width   : 16px;
+    margin  : 0;
+    padding : 0;
+    appearance:none;
+    -moz-appearance:none; /* Firefox */
+    -webkit-appearance:none; /* Safari and Chrome */
   }
-}
 
-.certificate-example-box {
-  padding: 30px 0;
-  margin: 0 40px;
-
-  img {
-    outline: $gray-color6 solid 2px;
+  input[type=checkbox] + label,
+  input[type=radio   ] + label {
+    display      : inline-block;
+    margin-left  : -20px;
+    padding-left : 28px;
+    line-height  : 16px;
+    height: 34px;
   }
-}
-
-/*****************/
-/* Form Elements */
-/*****************/
 
-input[type=checkbox],
-input[type=radio   ] {
-  width   : 16px;
-  margin  : 0;
-  padding : 0;
-  appearance:none;
-  -moz-appearance:none; /* Firefox */
-  -webkit-appearance:none; /* Safari and Chrome */
-}
+  input[type=radio   ] + label {
+    @include image("general/radio.svg", 16px);
 
-input[type=checkbox] + label,
-input[type=radio   ] + label {
-  display      : inline-block;
-  margin-left  : -20px;
-  padding-left : 28px;
-  line-height  : 16px;
-  height: 34px;
-}
-
-input[type=radio   ] + label {
-  @include image("general/radio.svg", 16px);
+    .other & {
+      background-position: left center;
+    }
 
-  .other & {
-    background-position: left center;
+    &:hover {
+      background-image: url("../images/general/radio-hover.svg");
+    }
   }
 
-  &:hover {
-    background-image: url("../images/general/radio-hover.svg");
+  input[type=radio   ]:checked  + label {
+    background-image: url("../images/general/radio-checked.svg");
   }
-}
-
-input[type=radio   ]:checked  + label {
-  background-image: url("../images/general/radio-checked.svg");
-}
 
-input[type=checkbox] + label {
-  @include image("general/checkbox.svg", 16px);
+  input[type=checkbox] + label {
+    @include image("general/checkbox.svg", 16px);
 
-  &:hover {
-    @include image("general/checkbox-hover.svg", 16px);
+    &:hover {
+      @include image("general/checkbox-hover.svg", 16px);
+    }
   }
-}
 
-input[type=checkbox]:checked + label {
-  @include image("general/checkbox-checked.svg", 16px);
-}
+  input[type=checkbox]:checked + label {
+    @include image("general/checkbox-checked.svg", 16px);
+  }
 
 
-.btn {
-  border-radius: 0;
-  border: none;
-  text-align: center;
-  color: $white;
-  width: auto;
-  font-size: 30px;
-  height: 50px;
-  line-height: 50px;
-  padding: 0 10%;
+  .btn {
+    border-radius: 0;
+    border: none;
+    text-align: center;
+    color: $white;
+    width: auto;
+    font-size: 30px;
+    height: 50px;
+    line-height: 50px;
+    padding: 0 10%;
+  }
 }
 
-.icount-iframe {
-  height: 620px;
+#donate-process-thank-you {
+  p {
+    font-size: 22px;
+    line-height: 46px;
+  }
 }
 
 /**************/
 /* responsive */
 /**************/
 
-@import "responsive";
-
-@media (max-width: 1199px) {
-
-}
+// @import "responsive";
```

Comment:
Something doesn't seem OK here: although you moved all responsive data to _responsive.scss, you still don't access _responsive.scss as you commented this line.
In addition, please remove all commented definitions below this line.